ROADMAP.md

```diff
@@ -0,0 +1,95 @@
+# Roadmap
+
+## Overview
+
+At Ratify, our mission is to safeguard the container supply chain by ratifying trustworthy and compliant artifacts. We achieve this through a robust and pluggable verification engine that includes built-in verifiers. These verifiers can be customized to validate supply chain metadata associated with artifacts, covering essential aspects such as signatures and attestations (including vulnerability reports, SBOM, provenance data, and VEX documents). As the landscape of supply chain security evolves, we actively develop new verifiers, which can be seamlessly integrated into our verification engine. Additionally, if you have a specific use case, you can create your own verifier following our comprehensive guidance. Each verifier will generate detailed verfication reports, which can be consumed by various policy controllers to enforce policies.
+
+Ratify serves several major scenarios. It seamlessly integrates with OPA Gatekeeper as the Kubernetes policy controller. By doing so, it safeguards your Kubernetes cluster against untrustworthy and non-compliant container images. Ratify functions as an external data provider for Gatekeeper, supplying artifact verification results that align with defined policies. Furthermore, Ratify can be employed at container runtime via the plugin interface, enabling policy evaluation based on its artifact verification outcomes. This enables more granular artifact verification at the Kubernetes node level. Last but not least, Ratify can be integrated into your CI/CD pipeline to ensure the trustworthiness and compliance of container images before using them. 
```

Comment:
> Ratify can be employed at container runtime via the plugin interface, enabling policy evaluation based on its artifact verification outcomes. This enables more granular artifact verification at the Kubernetes node level. Last but not least, Ratify can be integrated into your CI/CD pipeline to ensure the trustworthiness and compliance of container images before using them.

Should we describe these two functionalities as "designed to be" because these two are planned but not supported yet at this moment?